views/pages/user-detail.ejs

```diff
@@ -9,8 +9,9 @@
   <header>
     <%- include('../partials/header'); %>
   </header>
-
+  
   <main class="user-detail">
+    <h1 class="title"> Gebruiker details</h1>
```

Comment:
Je mag niet twee keer op 1 pagina een <h1> plaatsen. In dit geval maakt het niet zoveel uit, want ik heb de hele user-detail pagina veranderd. Let hier de volgende keer op